paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/SortCompactActionForUnawareBucketITCase.java

```diff
@@ -259,22 +261,26 @@ public void testRandomSuffixWorks() throws Exception {
     }
 
     private void zorder(List<String> columns) throws Exception {
+        String rangeStrategy = RANDOM.nextBoolean() ? "size" : "quantity";
         if (RANDOM.nextBoolean()) {
-            createAction("zorder", columns).run();
+            createAction("zorder", rangeStrategy, columns).run();
         } else {
-            callProcedure("zorder", columns);
+            callProcedure("zorder", rangeStrategy, columns);
```

Comment:
I think we need not to add rangStrategy in the parameters. We can find anywhere in the sort process.

Reply:
For example, we can find in SortUtils.sortStreamByKey, `Strategy strategy = table.coreOptions().sortRangeStrategy`